frontend/src/Admin/components/AssignJudgesTable.js

```diff
@@ -133,7 +134,7 @@ class AssignJudgesTable extends Component {
           <ModalBody>
             <p>
               Removing a judge will permanently delete any votes they have made
-              in this show.
+              in this {showId ? "show" : "portfolio period"}.
```

Comment:
I'm using the same component that the first team built. This was the only show specific line in it.